backend/django/core/migrations/0068_auto_20220617_1413.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.9 on 2022-06-17 14:13
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
So this change looks a bit different. Is it possible there is another migration file? When I ran the command, I got a migration with the charfield change like below

```
class Migration(migrations.Migration):

    dependencies = [
        ('core', '0068_auto_20220617_1413'),
    ]

    operations = [
        migrations.AlterField(
            model_name='externaldatabase',
            name='export_schema',
            field=models.CharField(max_length=1024, null=True),
        ),
        migrations.AlterField(
            model_name='externaldatabase',
            name='export_table_name',
            field=models.CharField(max_length=1024, null=True),
        ),
    ]
```